src/main/java/bio/terra/service/filedata/azure/util/BlobContainerCopier.java

```diff
@@ -22,6 +38,9 @@ public class BlobContainerCopier {
   private static final Logger logger = LoggerFactory.getLogger(BlobContainerCopier.class);
   private static final Duration DEFAULT_SAS_TOKEN_EXPIRATION = Duration.ofHours(24);
   private final BlobContainerClientFactory destinationClientFactory;
+  private static final String USER_PROJECT_QUERY_PARAM = "userProject";
+  // Signing URL for 48 hours to handle large file transfers
+  private static final int SIGNED_URL_DURATION_MINUTES = 60 * 48;
```

Comment:
You can use `TimeUnit` to handle `* 60` for you
```suggestion
  private static final long SIGNED_URL_DURATION_MINUTES = TimeUnit.HOURS.toMinutes(48);
```
Or use 48 here and pass in `TimeUnit.HOURS` below.

Reply:
good call.  (sadly I can't change the signature of the method below)